cmd/karmadactl/karmadactl.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package main
 
 import (
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
Blank Import can present some challenges for subsequent maintenance, can you help add some short comments to show the reason we added it?

Reply:
when we add the Blank Import, the init function of the oidc package will be executed, it will register a oidc AuthProviderPlugin, the plugin help us to connect k8s clusters use oidc  mode
`
func init() {
	if err := restclient.RegisterAuthProviderPlugin("oidc", newOIDCAuthProvider); err != nil {
		klog.Fatalf("Failed to register oidc auth plugin: %v", err)
	}
}
`

Reply:
You can add this to the comments of the code:

```go
_ "k8s.io/client-go/plugin/pkg/client/auth/oidc" //  Register AuthProviderPlugin, using OIDC mode to connect to Kubernetes
```

Reply:
ok